<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-VKAG1RPA" failures="2" tests="16" name="Type" time="101.421" errors="0" timestamp="2021-02-05T11:29:35 IST">
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="selectALL_forTypeCode" time="4.18"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="createTypeCode" time="3.626"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="createTypeCode" time="0.96"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="createTypeCode" time="1.265"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="saveTypeCode" time="24.894">
    <failure type="java.lang.AssertionError" message="No Error Message validation for save action expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: No Error Message validation for save action expected [true] but found [false]
at com.extraPOSTest.testCases.inventory.maintenance.TC_Type.saveTypeCode(TC_Type.java:282)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- saveTypeCode -->
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="addTwoSameTypeCode" time="1.667"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="addTwoSameTypeCode" time="0.783"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="saveTwoTypeCode" time="5.146"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="searchTypePage" time="3.444"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="searchTypePage" time="3.524"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="searchTypePage" time="3.297"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="searchTypePage" time="4.077"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="checkReverseOrderValidation_TypeCode" time="2.047"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="checkReverseOrderValidation_TypeCode" time="0.576"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="checkReverseOrderValidation_TypeCode" time="0.308"/>
  <testcase classname="com.extraPOSTest.testCases.inventory.maintenance.TC_Type" name="checkReverseOrderValidation_TypeCode" time="21.374">
    <failure type="java.lang.AssertionError" message="No validation for duplicate Typecode expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: No validation for duplicate Typecode expected [true] but found [false]
at com.extraPOSTest.testCases.inventory.maintenance.TC_Type.checkReverseOrderValidation_TypeCode(TC_Type.java:818)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- checkReverseOrderValidation_TypeCode -->
</testsuite> <!-- Type -->
